config.go

```diff
@@ -50,8 +50,10 @@ const (
 	defaultMaxLogFiles        = 3
 	defaultMaxLogFileSize     = 10
 
-	defaultTorDNS       = "soa.nodes.lightning.directory:53"
-	defaultTorSOCKSPort = 9050
+	defaultTorDNS                = "soa.nodes.lightning.directory:53"
+	defaultTorSOCKSPort          = 9050
+	defaultTorControlPort        = 9051
+	defaultTorPrivateKeyFilename = "onion_private_key"
```

Comment:
May want to prefix this with `v2` (the variable name), and also the file name since a future version might let the user specify their _own_ private key when we extend to also support auto `v3` over the control port.

Reply:
Fixed.